src/main/kotlin/xyz/theprogramsrc/simplecoreapi/global/utils/update/GitHubUpdateChecker.kt

```diff
@@ -13,22 +13,21 @@ import java.time.format.DateTimeFormatter
  * @param repo The repository to check
```

Comment:
Maybe add that the repo needs to be formatted with `User/Folder`, some people might think is just the repository name